pkg/reconciler/core/logicalclusterdeletion/deletion/logicalcluster_resource_deletor.go

```diff
@@ -367,6 +367,7 @@ func (d *logicalClusterResourcesDeleter) deleteAllContent(ctx context.Context, w
 		// Don't try to delete projected resources such as tenancy.kcp.dev v1beta1 Workspaces - these are virtual
 		// projections and we shouldn't try to delete them. The projections will disappear when the real underlying
 		// data (e.g. ClusterWorkspaces) are deleted.
+		// TODO (fgiloux): does the example (workspace as a virtual projection) need to be changed?
```

Comment:
Should the comment get updated?

Reply:
we still have the projection, just inverted it. But it will be removed soon.